cypress/integration/smoketests.ts

```diff
@@ -0,0 +1,61 @@
+describe('The Home Page', () => {
+  it('has a working comparator modal', () => {
+    cy.visit('/')
+
+    cy.get('[id^=downshift-0-input]')
+      .click()
+      .type('testing-library/dom-testing-library')
+    cy.wait(5000)
+    cy.contains('p', 'testing-library/dom-testing-library').click()
+
+    // TODO: make selecting first version specific to the first selector box
+
+    cy.contains('select', 'Choose a release').select('v6.16.0')
+    cy.url().should(
+      'eq',
+      'http://localhost:3000/?repo=testing-library%2Fdom-testing-library&from=v6.16.0'
+    )
+
+    // TODO: Select v8.1.0 from second version dropdown box
+  })
+})
+
+describe('Comparator smoketest', () => {
+  it('with standard query string shows expected results', () => {
```

Comment:
This test is basically trying to figure out if we show all the expected items from the standard query string.